docs/faq.rst

```diff
@@ -13,7 +13,7 @@ Yes, the `deduplication`_ technique used by
 Also, we have optional simple sparse file support for extract.
 
 If you use non-snapshotting backup tools like Borg to back up virtual machines,
-then these should be turned off for doing so. Backing up live VMs this way can (and will)
+then the VMs should be turned off before doing so. Backing up live VMs can (and will)
```

Comment:
You can totally do a block-level snapshot of a running VM, it's just that this gives crash-consistency which can be anywhere from annoying to disastrous when restoring.